src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -525,9 +530,11 @@ public OpenIdConnectAuthenticationHandler(HttpClient backchannel)
             {
                 return null;
             }
+            message = securityTokenReceivedContext.ProtocolMessage;
+            var idToken = securityTokenReceivedContext.SecurityToken;
 
```

Comment:
SecurityToken is far too generic. And actually, I wonder if it's not the access token.